### ISSUE TITLE
[Bug] 富文本标签不响应state 

### ISSUE THREAD

**Issue by skie1997** [VisActor/VChart#3465]
### Version

1.12.13

### Link to Minimal Reproduction

null

### Steps to Reproduce

```ts
const iconUrl = week => `https://lf9-dp-fe-cms-tos.byteorg.com/obj/bit-cloud/${week}-icon-vchart-demo.svg`;
const iconMap = {
  Monday: iconUrl('Monday'),
  Tuesday: iconUrl('Tuesday'),
  Wednesday: iconUrl('Wednesday'),
  Thursday: iconUrl('Thursday'),
  Friday: iconUrl('Friday')
};

const spec = {
  type: 'bar',
  width: 835,
  height: 520,
  data: [
    {
      id: 'barData',
      values: [
        { day: 'Monday', sales: 22 },
        { day: 'Tuesday', sales: 13 },
        { day: 'Wednesday', sales: 25 },
        { day: 'Thursday', sales: 29 },
        { day: 'Friday', sales: 38 }
      ]
    }
  ],
  label: {
    visible: true,
    position: 'top',
    interactive: true,
    id: 'label',
    // formatMethod: (value, data) => {
    //   return {
    //     type: 'rich',
    //     text: [
    //       {
    //         image: iconMap[data.day],
    //         width: 18,
    //         height: 18,
    //         fontWeight: 400
    //       },
    //       {
    //         text: ` ${data.day}`,
    //         fontSize: 12,
    //         underline: true
    //       },
    //       {
    //         text: `: `,
    //         fontSize: 12
    //       },
    //       {
    //         text: `${value} `,
    //         fontSize: 20,
    //         fontStyle: 'italic',
    //         fill: 'black',
    //         // fontWeight: 800
    //       }
    //     ]
    //   };
    // },
    textType: 'rich',
    style: {
      text: [
        {
            text: `xxxxx`,
            fontSize: 12
          },
      ],
      opacity: 1
    },
    state: {
      hover: {
        opacity: 1,
        // fill: 'red',
      },
      hover_reverse: {
        opacity: 0.2,
      }
    }
  },
  xField: 'day',
  yField: 'sales',
  seriesField: 'day'
};

const vchart = new VChart(spec, { dom: CONTAINER_ID });
vchart.renderSync();

// Just for the convenience of console debugging, DO NOT COPY!
window['vchart'] = vchart;
```

### Current Behavior

![20241126215931_rec_](https://github.com/user-attachments/assets/915edeeb-04c0-4986-aa90-9a9275401a59)

### Expected Behavior

![20241126220003_rec_](https://github.com/user-attachments/assets/48bc6d95-dd98-4ad6-8b52-784efaad75a0)

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by xile611** [comment]
vrender 底层富文本更新透明度有问题：

```
const graphics = [];
const rich = VRender.createRichText({
    x: 0,
    y: 0,
    width: 300,
    height: 0,
    textConfig: [
      {
        text: '即将发布的',
        fill: '#000',
        fontSize: 16,
        textDecoration: 'line-through'
      },
      {
        text: 'VisActor',
        fontWeight: 'bold',
        fontSize: 30,
        fill: '#0013e6'
      },
      {
        text: '由数据而生，演绎数据之美。致力于成为面向叙事的智能可视化解决方案。截至目前，',
        fill: '#000',
        fontSize: 16,
        lineHeight: 26
      },
      {
        text: 'VisActor',
        fontStyle: 'italic',
        fill: '#000',
        fontSize: 16,
        lineHeight: 26,
        textDecoration: 'underline'
      },
      {
        text: '已经开源了多个可视化模块，它们分别是\n',
        fill: '#000',
        fontSize: 16,
        lineHeight: 26
      },
      {
        text: 'VChart 故事讲述者',
        fill: '#3f51b5',
        fontSize: 16,
        lineHeight: 26,
        fontWeight: 'bold'
      },
      {
        text: '[1]',
        script: 'super',
        fill: '#000',
        fontSize: 16,
        lineHeight: 26,
        fontWeight: 'bold'
      },
      {
        text: '。它不只是开箱即用的多端图表库，更是生动灵活的数据故事讲述者。\n',
        fill: '#000',
        fontSize: 16,
        lineHeight: 26
      },
      {
        text: 'VTable 方格艺术家',
        fill: '#3f51b5',
        fontSize: 16,
        lineHeight: 26,
        fontWeight: 'bold'
      },
      {
        text: '[2]',
        script: 'super',
        fill: '#000',
        fontSize: 16,
        lineHeight: 26,
        fontWeight: 'bold'
      },
      {
        text: '。它不只是一款高性能的多维数据分析表格，更是一个在行列间创作的方格艺术家。\n',
        fill: '#000',
        fontSize: 16,
        lineHeight: 26
      },
      {
        text: 'VGrammar 数据魔法师',
        fill: '#3f51b5',
        fontSize: 16,
        lineHeight: 26,
        fontWeight: 'bold'
      },
      {
        text: '[3]',
        script: 'super',
        fill: '#000',
        fontSize: 16,
        lineHeight: 26,
        fontWeight: 'bold'
      },
      {
        text: '。不只是生成万千图表的可视化语法，更是化枯燥为神奇的数据魔法师。\n',
        fill: '#000',
        fontSize: 16,
        lineHeight: 26
      },
      {
        text: 'VRender 可视化渲染',
        fill: '#3f51b5',
        fontSize: 16,
        lineHeight: 26,
        fontWeight: 'bold'
      },
      {
        text: '[4]',
        script: 'super',
        fill: '#000',
        fontSize: 16,
        lineHeight: 26,
        fontWeight: 'bold'
      },
      {
        text: '。不只是一个功能丰富的可视化渲染擎，更是一支得心应手的生花妙笔。',
        fill: '#000',
        fontSize: 16,
        lineHeight: 26
      }
    ],
    opacity: 1
  });
graphics.push(
  rich
);
const stage = new Stage({
  container: CONTAINER_ID,
  autoRender: true
});

graphics.forEach(g => {
  stage.defaultLayer.add(g);
});

rich.setAttributes({ opacity: 0.2 })
stage.render();

window.stage = stage;
```

**Comment by skie1997** [comment]
预期修复时间：两周左右，修复好后可以先发alpha版本接入测试

**Comment by xile611** [comment]
fixed in 1.12.14
